servers/rendering/renderer_rd/environment/gi.cpp

```diff
@@ -3322,7 +3322,11 @@ void GI::init(RendererSceneSkyRD *p_sky) {
 				RD::Uniform u;
 				u.uniform_type = RD::UNIFORM_TYPE_TEXTURE;
 				u.binding = 0;
-				u.append_id(texture_storage->texture_rd_get_default(RendererRD::DEFAULT_RD_TEXTURE_CUBEMAP_WHITE));
+				if (p_sky->sky_use_cubemap_array) {
+					u.append_id(texture_storage->texture_rd_get_default(RendererRD::DEFAULT_RD_TEXTURE_CUBEMAP_ARRAY_BLACK));
```

Comment:
You implemented `DEFAULT_RD_TEXTURE_CUBEMAP_ARRAY_WHITE` but are using `DEFAULT_RD_TEXTURE_CUBEMAP_ARRAY_BLACK`, is that intended?

Reply:
Nope... Copy-paste mistake. Fixed.